### PR TITLE
[Blog] week 27

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: content/posts/July-2020.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: "July 2020"
+date: 2020-07-03
+draft: false
+---
+
+## Week 27 (June 29th - July 3rd)
+
+### packit
+
+* We introduced a new command `packit validate-config` which validates the contect of a package config
```
Comment (csomh) [inline content/posts/July-2020.md:11]:
```suggestion
* We introduced a new command `packit validate-config` to validate the content of a package configuration files
```

Comment (TomasTomecek) [reply content/posts/July-2020.md:11]: I'd even say 

    `packit.yaml` in your project

File: content/posts/July-2020.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  (contributed by [@TomasJani](https://github.com/TomasJani), [#826](https://github.com/packit-service/packit/pull/826)).
+
+
+### packit-as-a-service
+
+
+* You are now able to override configuration for builds, releases and other jobs (by [@TomasTomecek](https://github.com/TomasTomecek), [#676](https://github.com/packit-service/packit-service/pull/676)).
+ Read more about this feature [here](/docs/configuration/#overriding-global-parameters).
```
Comment (TomasTomecek) [inline content/posts/July-2020.md:19]: well done! linking to docs themselves

File: content/posts/July-2020.md (hunk 1 above)
Comment (lachmanfrantisek) [reply content/posts/July-2020.md:11]: But we support also different names:

```python
CONFIG_FILE_NAMES = [
    ".packit.yaml",
    ".packit.yml",
    ".packit.json",
    "packit.yaml",
    "packit.yml",
    "packit.json",
]
```

**Pull request merged**